validator/proposer/service.go

```diff
@@ -44,15 +52,36 @@ func (p *Proposer) Stop() error {
 }
 
 // run the main event loop that listens for a proposer assignment.
-func (p *Proposer) run(done <-chan struct{}) {
+func (p *Proposer) run(done <-chan struct{}, client pb.ProposerServiceClient) {
 	for {
 		select {
 		case <-done:
 			log.Debug("Proposer context closed, exiting goroutine")
 			return
+		// TODO: Instead subscribe to a proposal assignment feed that contains
+		// important fields from the currently received beacon block thta will be put into
+		// a proposal RPC message such as slot number and parent hash.
+		//
+		// TODO: On the beacon node side, calculate active and crystallized and update the
+		// active/crystallize state hash values in the proposed block.
 		case <-p.beaconService.ProposerAssignment():
 			log.Info("Performing proposer responsibility")
-			continue
+
+			// Sending empty values for now.
+			// TODO: Implement real proposals with randao reveals and attestation fields.
+			// TODO: Add timestamp, parent hash, and slot number.
+			req := &pb.ProposeRequest{
+				RandaoReveal:            []byte{},
```

Comment:
Rather than purposefully setting these to blank values, why not just omit them until it is implemented?

Or perhaps use some interesting values?